rotkehlchen/globaldb/upgrades/v2_v3.py

```diff
@@ -370,9 +371,9 @@ def translate_assets_in_price_table(cursor: 'DBCursor') -> list[tuple[str, str,
     return updated_rows
 
 
+@enter_exit_debug_log(name='GlobalDB v2->v3 upgrade')
 def migrate_to_v3(connection: 'DBConnection') -> None:
     """Upgrade assets information and migrate globaldb to version 3"""
-    log.debug('Entered globaldb v2->v3 upgrade')
```

Comment:
you also need to remove `log.debug('Finished globaldb v2->v3 upgrade')` from line 550